feature_server/map.py

```diff
@@ -111,10 +111,10 @@ def __init__(self, name = "pyspades"):
         self.name = name
 
     def get_seed(self):
-        if self.seed is None:
-            random.seed()
-            self.seed = random.randint(0, math.pow(2, 31))
-        return self.seed
+        if self.seed is not None:
+            return self.seed
+        random.seed()
```

Comment:
Thanks for spotting, but it is expected behaviour:

https://github.com/NateShoffner/PySnip/pull/6/commits/9ede53f82d52b89416841f48c02b729fd0249895
https://code.google.com/archive/p/pyspades/issues/282